target/linux/realtek/image/Makefile

```diff
@@ -26,6 +26,16 @@ define Device/Default
 	append-metadata | check-size
 endef
 
+# "NGE" refers to the uImage magic
+define Device/netgear_nge
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | uImage lzma
+  SOC := rtl8380
+  IMAGE_SIZE := 14848k
+  UIMAGE_MAGIC := 0x4e474520
+  DEVICE_VENDOR := Netgear
```

Comment:
Please use all-caps "NETGEAR" here, as we consistently do in the entire tree.

Reply:
To be clear, ```ZyXEL``` and ```D-Link``` use camel case in the other device definitions in this Makefile.  I'm fine changing it to uppercase if that is the general policy.